app/src/main/res/layout/item_tracker_network_header.xml

```diff
@@ -45,12 +45,27 @@
 
     <ImageView
         android:id="@+id/icon"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="24dp"
+        android:layout_height="24dp"
         android:importantForAccessibility="no"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         tools:src="@drawable/network_pill_facebook_inc" />
 
+    <TextView
+        android:id="@+id/unknownIcon"
+        android:layout_width="24dp"
```

Comment:
I'm most probably wrong, but should this be `22dp` too?

Reply:
We talked about this, the design calls for 24dp here.